diff --git a/node_modules/@apollosproject/ui-connected/src/ConnectScreenConnected/index.js b/node_modules/@apollosproject/ui-connected/src/ConnectScreenConnected/index.js
index 521d8ed..7069309 100644
--- a/node_modules/@apollosproject/ui-connected/src/ConnectScreenConnected/index.js
+++ b/node_modules/@apollosproject/ui-connected/src/ConnectScreenConnected/index.js
@@ -60,7 +60,7 @@ const ConnectScreenConnected = (props) => {
             <SuggestedFollowListConnected
               refetchRef={refetchRef}
               onPressFollowListButton={() => setModalOpen(true)}
-              followListButtonTitle={'Find People to Follow'}
+              followListButtonTitle={'Explore Connections'}
             />
 
             {ActionTable && <ActionTable />}
diff --git a/node_modules/@apollosproject/ui-connected/src/ContentNodeConnected/ContentTitlesConnected.js b/node_modules/@apollosproject/ui-connected/src/ContentNodeConnected/ContentTitlesConnected.js
index 5f1b8f4..3c96d3f 100644
--- a/node_modules/@apollosproject/ui-connected/src/ContentNodeConnected/ContentTitlesConnected.js
+++ b/node_modules/@apollosproject/ui-connected/src/ContentNodeConnected/ContentTitlesConnected.js
@@ -8,6 +8,7 @@ import { useShare } from '../ShareButtonConnected';
 const ContentTitlesConnected = ({ node }) => {
   const [isLiked, like] = useLike(node?.id);
   const share = useShare(node?.id);
+  console.log("ContentTitlesConnected node", share);
   return (
     <ContentTitles
       title={node?.title}
diff --git a/node_modules/@apollosproject/ui-connected/src/EventDetailsConnected/getEvent.js b/node_modules/@apollosproject/ui-connected/src/EventDetailsConnected/getEvent.js
index 6c7f703..c2e5a03 100644
--- a/node_modules/@apollosproject/ui-connected/src/EventDetailsConnected/getEvent.js
+++ b/node_modules/@apollosproject/ui-connected/src/EventDetailsConnected/getEvent.js
@@ -14,6 +14,19 @@ export const EVENT_FRAGMENT = gql`
       }
     }
   }
+  fragment groupFragment on Group {
+    id
+    title
+    location
+    start
+    end
+    htmlContent
+    coverImage {
+      sources {
+        uri
+      }
+    }
+  }
 `;
 
 export default gql`
@@ -23,6 +36,9 @@ export default gql`
       ... on Event {
         ...eventFragment
       }
+      ... on Group{
+        ...groupFragment
+      }
     }
   }
   ${EVENT_FRAGMENT}
diff --git a/node_modules/@apollosproject/ui-connected/src/EventDetailsConnected/index.js b/node_modules/@apollosproject/ui-connected/src/EventDetailsConnected/index.js
index 2c4b2a4..853fd95 100644
--- a/node_modules/@apollosproject/ui-connected/src/EventDetailsConnected/index.js
+++ b/node_modules/@apollosproject/ui-connected/src/EventDetailsConnected/index.js
@@ -36,8 +36,8 @@ const EventDetailsConnected = ({ nodeId }) => {
     fetchPolicy: 'cache-and-network',
   });
   const event = data?.node;
-
-  return event && event.__typename === 'Event' ? (
+  return event && (event.__typename === 'Event' || event.__typename === 'Group') ? (
+    console.log("EVENT start IS >>>>>> ", event?.location) ,
     <Paragraph>
       {event.start ? (
         <EventInfoItem
diff --git a/node_modules/@apollosproject/ui-connected/src/FollowListConnected/RequestedFollowListConnected.js b/node_modules/@apollosproject/ui-connected/src/FollowListConnected/RequestedFollowListConnected.js
index cac25e8..6b1c74e 100644
--- a/node_modules/@apollosproject/ui-connected/src/FollowListConnected/RequestedFollowListConnected.js
+++ b/node_modules/@apollosproject/ui-connected/src/FollowListConnected/RequestedFollowListConnected.js
@@ -49,7 +49,7 @@ RequestedFollowListConnected.propTypes = {
 
 RequestedFollowListConnected.defaultProps = {
   Component: FollowListConnected,
-  Header: () => <FeatureTitles subtitle={'Follow Requests'} />, // eslint-disable-line react/display-name
+  Header: () => <FeatureTitles subtitle={'Connection Requests'} />, // eslint-disable-line react/display-name
 };
 
 RequestedFollowListConnected.displayName = 'SuggestedFollowListConnected';
diff --git a/node_modules/@apollosproject/ui-connected/src/FollowListConnected/SuggestedFollowListConnected.js b/node_modules/@apollosproject/ui-connected/src/FollowListConnected/SuggestedFollowListConnected.js
index c3c8e55..4ec24cc 100644
--- a/node_modules/@apollosproject/ui-connected/src/FollowListConnected/SuggestedFollowListConnected.js
+++ b/node_modules/@apollosproject/ui-connected/src/FollowListConnected/SuggestedFollowListConnected.js
@@ -49,7 +49,7 @@ SuggestedFollowListConnected.propTypes = {
 
 SuggestedFollowListConnected.defaultProps = {
   Component: FollowListConnected,
-  Header: () => <FeatureTitles subtitle={'People to Follow'} />, // eslint-disable-line react/display-name
+  Header: () => <FeatureTitles subtitle={'Connect with Congregation'} />, // eslint-disable-line react/display-name
 };
 
 SuggestedFollowListConnected.displayName = 'SuggestedFollowListConnected';
diff --git a/node_modules/@apollosproject/ui-connected/src/FollowPeopleFeature/index.js b/node_modules/@apollosproject/ui-connected/src/FollowPeopleFeature/index.js
index 2183e23..0664a59 100644
--- a/node_modules/@apollosproject/ui-connected/src/FollowPeopleFeature/index.js
+++ b/node_modules/@apollosproject/ui-connected/src/FollowPeopleFeature/index.js
@@ -15,7 +15,7 @@ const FollowPeopleFeature = ({ refetchRef }) => {
       <SuggestedFollowListConnected
         refetchRef={refetchRef}
         onPressFollowListButton={() => setModalOpen(true)}
-        followListButtonTitle={'Find People to Follow'}
+        followListButtonTitle={'Explore Connections'}
       />
       <FollowListSearchModalConnected
         open={modalOpen}
